server.py

```diff
@@ -226,6 +244,27 @@ def do_get_responses():
         return jsonify({}), 404
 
 
+@app.route('/feedback/<feedback_id>', methods=['GET'])
+def do_get_feedback(feedback_id):
+    try:
+        int(feedback_id)
+    except ValueError:
+        raise InvalidUsageError("feedback_id supplied is not a valid id", 400)
+
+    result = get_feedback(feedback_id=feedback_id)
+
+    if result:
+        try:
+            response = jsonify(result[0].data)
+            response.headers['Content-MD5'] = hashlib.md5(response.data).hexdigest()
+            return response
+        except IndexError:
+            logger.exception('Empty items list in result.')
```

Comment:
I don't think exception is the right level here - should we really be potentially alerting on this? Info makes more sense